src/main/java/net/datafaker/providers/base/Number.java

```diff
@@ -52,12 +52,18 @@ public int numberBetween(int min, int max) {
         if (min == max) return min;
         final int realMin = Math.min(min, max);
         final int realMax = Math.max(min, max);
-        if (realMax - realMin >= realMin && (realMin >= 0 || realMax - realMin >= realMax)) {
+        if (isValidRange(realMin, realMax)) {
             return faker.random().nextInt(realMax - realMin) + realMin;
         }
         return decimalBetween(realMin, realMax).intValue();
     }
 
+   
+    private boolean isValidRange(int realMin, int realMax) {
+        return realMax - realMin >= realMin && (realMin >= 0 || realMax - realMin >= realMax);
```

Comment:
Probably `realMax - realMin` could be extracted into a var called e.g. `amplitude`

Reply:
> Probably `realMax - realMin` could be extracted into a var called e.g. `amplitude`

Changed.